samples/caf_sensor_manager/configuration/common/event_proxy_init.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#if CONFIG_EVENT_MANAGER_PROXY
+	int init_event_proxy(char *proxy_subscribe_event);
+#else
+	static inline int init_event_proxy(char *proxy_subscribe_event)
+		{
+		return 0;
```

Comment:
indent is wrong

Reply:
I personally would prefer to not indent it additionally. Also for the function declaration (line 8)